spec/integration/ldap_spec.rb

```diff
@@ -64,38 +65,39 @@
         :owner => 'uid=quentin,ou=people,dc=example,dc=org',
         :uniquemember => ['uid=samuel', 'uid=uma', 'uid=john']
       }
-      Hydra::LDAP.create_group('PulpFiction', attrs).should be_true
+      expect(Hydra::LDAP.create_group('PulpFiction', attrs)).to be true
     end
 
     after do
-      Hydra::LDAP.delete_group('PulpFiction').should be_true
+      Hydra::LDAP.delete_group('PulpFiction')
     end
 
     it "should return a list of groups owned by quentin" do
       attrs = {
-        :cn => 'TR',
+        :cn => 'TrueRomance',
```

Comment:
I read the test a couple of times wondering if I was missing something -- maybe it never passed?  On that score, I can add a .travis.yml file as long as I'm adding a new commit to fix your other comment.